packages/env/lib/init-config.js

```diff
@@ -6,14 +6,13 @@ const path = require( 'path' );
 const { writeFile, mkdir } = require( 'fs' ).promises;
 const { existsSync } = require( 'fs' );
 const yaml = require( 'js-yaml' );
-const os = require( 'os' );
 
 /**
  * Internal dependencies
  */
 const { loadConfig } = require( './config' );
 const buildDockerComposeConfig = require( './build-docker-compose-config' );
-
+const { ValidationError } = require( './config' );
```

Comment:
This can be combined with the above `require()` (shouldn't the linter have caught this?)